brave/browser-laptop
# Add button to restart tor without restarting brave

fix https://github.com/brave/browser-laptop/issues/14567

TEST PLAN:
1. start a local http server on port 9250, ex: "python3 -m http.server 9250 --bind 127.0.0.1"
2. start brave and open tor tab
3. wait for tor error to show
4. click 'retry' button
5. it should show 'connecting...' then show the error again
6. now kill the HTTP server from step 1
7. click 'retry' again. tor should connect this time.

## Submitter Checklist:


## Test Plan:


## Reviewer Checklist:


Tests
